docs/themes/porter/layouts/index.html

```diff
@@ -49,21 +52,29 @@ <h1>
 
         <div id="helm" class="home">
 
-          <section class="billboard">
-            <div class="container">
-              <h1>Your application and everything you need to deploy<br/>Together 💝</h1>
-              <h2>Package your application artifact, client tools, configuration and deployment logic together as a versioned bundle that you can distribute, and then install with a single command</h2>
+          <section class="billboard row mx-0">
+            <div class="container col-md-10 text-center">
+              <h1 class="mb-3">Your Application and everything you need to, <span id="underline1">Deploy Together </span> </h1>
```

Comment:
Can you change this to the following (per my original comment)? When I tried this locally, the underline seems to be absolutely positioned and isn't aligning with the span'd word.

```suggestion
              <h1 class="mb-3">Your Application and everything you need to deploy, <span id="underline1">together </span> </h1>
```

Reply:
Ok sure, I might have missed that part. I'll change that tomorrow morning. If you have any other feedback as well, do mention that, I'll take a look in the morning and make the required changes.